.circleci/config.yml

```diff
@@ -154,20 +154,55 @@ jobs:
           no_output_timeout: 1m
       - run:
           name: Run backward compatibility test
-          command: ./bin/run-backward-compatibility-tests
+          command: bb test back-compat
           no_output_timeout: 5m
       - save_cache:
           key: deps-{{ checksum "deps.edn" }}
           paths:
             - /home/circleci/.m2
+  integration-test:
+    executor: tools/clojurecli
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+      - run:
+          name: Run integration test
+          command: bb test integration
+          no_output_timeout: 5m
+      - save_cache:
+          key: deps-{{ checksum "deps.edn" }}
+          paths:
+            - /home/circleci/.m2
+  build:
+    executor: tools/clojurecli
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+      - run:
+          name: Build jar
+          command: bb jar
+      - persist_to_workspace:
+          root: /home/circleci/
+          paths:
+            - .m2
+            - .npm
+            - replikativ
   deploy:
     executor: tools/clojurecli
     steps:
       - attach_workspace:
           at: /home/circleci
       - run:
           name: deploy
-          command: clojure -T:build deploy
+          command: bb deploy
+  release:
+    executor: tools/clojurecli
+    steps:
+      - attach_workspace:
+          at: /home/circleci
+      - run:
+          name: release
+          command: bb release
```

Comment:
That was due to a typo in the bb.edn. Should work now.

Reply:
Tried to find the problem but this still fails for me...You can run it locally with `bb release` and it should show you an NPE pointing to `gh/overwrite-asset` [like here](https://app.circleci.com/pipelines/github/TimoKramer/datahike/431/workflows/4068b454-a0e9-499a-a999-c4c66a465575/jobs/2002/parallel-runs/0/steps/0-102).

I tried a few things but one thing I am pretty sure of is that you will need to specify `gh-release-artifact` in the deps of your bb.edn. but this unfortunately does not solve the problem:(

Reply:
I have specified `gh-release-artifact` in `bb/deps.edn` already. I think this is what you mean. For me it failed because I changed the keyword for the lib symbol and forgot this location apparently. Now it only gets stuck with an "access denied" for the http request. I am not sure atm if I actually don't have the permission to release or if there is still a problem with the code.

Reply:
it seems to work fine now on the server, it still fails with a NPE on my machine and I am lost on why...